website/docs/language/modules/develop/index.mdx

```diff
@@ -7,7 +7,7 @@ description: >-
 
 # Creating Modules
 
-> **Hands-on:** Try the [Reuse Configuration with Modules](https://learn.hashicorp.com/collections/terraform/modules?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn.
+> **Hands-on:** Try the [Reuse Configuration with Modules](https://learn.hashicorp.com/collections/terraform/modules?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorials.
 
 A _module_ is a container for multiple resources that are used together.
 Modules can be used to create lightweight abstractions, so that you can
```

Comment:
I know the first bit is passive but I'm hesitant to change it because "are used" I think implies something SLIGHTLY different than what you wanted to change it to. I would have to think more about this before rewording. BUT I will take the second edit - thank you!